Source/AaxDecrypter/NetworkFileStream.cs

```diff
@@ -110,10 +115,14 @@ public NetworkFileStream(string saveFilePath, Uri uri, long writePosition = 0, D
 		/// <summary> Update the <see cref="Dinah.Core.IO.JsonFilePersister{T}"/>. </summary>
 		private void OnUpdate()
 		{
-			RequestHeaders["Range"] = $"bytes={WritePosition}-";
 			try
 			{
-				Updated?.Invoke(this, EventArgs.Empty);
+				if (DateTime.UtcNow > NextUpdateTime)
+				{
+					Updated?.Invoke(this, EventArgs.Empty);
+					//JsonFilePersister Will not allow update intervals shorter than 100 milliseconds
+					NextUpdateTime = DateTime.UtcNow.AddMilliseconds(110);
+				}
```

Comment:
The recent change to JsonFilePersister created a throttle to the downloader. I spent hours trying to figure out why it was so slow!

Reply:
Oh shit, that might have been me.

Reply:
And I and reviewed it! I feel silly it took so long to realize what was happening.